app/models/person_with_remote_fields.rb

```diff
@@ -7,12 +7,11 @@ class PersonWithRemoteFields < Person
 
   attr_accessor *REMOTE_FIELDS
 
-
   def save(args = {})
```

Comment:
Since we're now extending Person, this could be an `after_save` callback instead of overwriting `save`. Thoughts?